### PR TITLE
Allow customized whitelists.

### PR DESCRIPTION
**Pull request by ygale**
Sometimes there is a need to customize the white list. Our specific use is that we are using the comments feature of (some recent version of) CKEditor. It inserts comments into the HTML using a non-standard HTML tag, so these are getting sanitized out.

This PR adds a new version of the `safeTags` filter, called `mySafeTags`, that allows you to supply custom versions of `safeTagName` and `sanitizeAttribute`. You can easily build your own version of these functions on top of the originals with small changes in behavior, such as adding or removing one or more specific items from the white list. Or you can build your own from scratch.

### CODE REVIEW

**Comment by ygale** [comment]
Sorry that the PR was a bit premature. It works now. And I added some tests.

**Comment by ygale** [comment]
@snoyberg Thanks! Done.

**Comment by ygale** [comment]
@snoyberg AppVeyor seems to be having some trouble, could you please take a look?

File: package.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name:        xss-sanitize
-version:     0.3.5.7
+version:     0.3.5.8
```
Comment (snoyberg) [inline package.yaml]: The given version bump is a patch version bump. We'd need a minor version bump to follow the PVP (bumping the C in A.B.C.D). In other words: 0.3.6.

Comment (ygale) [reply package.yaml]: Oops, sorry about that. Fixed.

**Pull request merged**